NachoSpigot-Server/src/main/java/net/minecraft/server/TileEntitySign.java

```diff
@@ -7,8 +7,8 @@
     public final IChatBaseComponent[] lines = new IChatBaseComponent[] { new ChatComponentText(""), new ChatComponentText(""), new ChatComponentText(""), new ChatComponentText("")};
     public int f = -1;
     public boolean isEditable = true;
-    private EntityHuman h;
```

Comment:
This is a dangerous change. If some NMS plugin tries to access `h` or `c` this will break.
Should we keep it? Maybe wrap it around a `WeakReference`?